test/gtest/ucp/test_ucp_am.cc

```diff
@@ -879,17 +881,14 @@ UCP_INSTANTIATE_TEST_CASE(test_ucp_am_nbx)
 
 
 class test_ucp_am_nbx_reply_always : public test_ucp_am_nbx {
-public:
-    bool am_lane_has_caps(uint64_t caps);
-
 protected:
     virtual unsigned get_send_flag() const
     {
         return UCP_AM_SEND_FLAG_REPLY;
     }
 };
 
-bool test_ucp_am_nbx_reply_always::am_lane_has_caps(uint64_t caps)
+bool test_ucp_am_nbx::am_lane_has_caps(uint64_t caps)
```

Comment:
move this func up

Reply:
Fixed in https://github.com/openucx/ucx/pull/9215